Jenkinsfile

```diff
@@ -12,7 +17,7 @@ try {
 
       if (jenkinsUserName != "admin" && env.BRANCH_NAME != "master") {
         try {
-          withCredentials([usernamePassword(credentialsId: 'github', usernameVariable: 'GITHUB_USERNAME', passwordVariable: 'GITHUB_TOKEN')]) {
+          withCredentials([usernamePassword(credentialsId: '5fe8ede9-bbdb-4803-a307-6924d4b4d9b5', usernameVariable: 'GITHUB_USERNAME', passwordVariable: 'GITHUB_TOKEN')]) {
```

Comment:
@mweisgut do you know anything about this change?

Reply:
Nope, no idea.

Reply:
Me neither

Reply:
Looking through Jenkins, I believe `GitHub` is correct.